packages/forms-web-app/src/routes/full-appeal/submit-appeal/planning-obligation-deadline.js

```diff
@@ -0,0 +1,23 @@
+const express = require('express');
+
+const router = express.Router();
+const planningObligationStatusController = require('../../../controllers/full-appeal/submit-appeal/planning-obligation-deadline');
```

Comment:
minor point, but this might be better named `planningObligationDeadlineController` to avoid confusion with `planning-obligation-status.js`